src/Microsoft.ML.FastTree/TreeEnsemble/TreeEnsemble.cs

```diff
@@ -357,9 +358,25 @@ internal void GetFeatureContributions(in VBuffer<float> features, ref VBuffer<fl
 
             builder.GetResult(ref contribs);
         }
+
+        public int GetMaxFeatureIndex()
```

Comment:
>GetMaxFeatureIndex [](start = 19, length = 18)

Need to have xml documentation for anything that is public.

Reply:
It's an internal class but of course, we should doc it.

---
In reply to: [252031398](https://github.com/dotnet/machinelearning/pull/2243#discussion_r252031398) [](ancestors = 252031398)